tailwind.config.js

```diff
@@ -8,7 +8,17 @@ module.exports = {
     ],
     darkMode: false, // or 'media' or 'class'
     theme: {
-        extend: {},
+        extend: {
+            typography: {
+                DEFAULT: {
+                    css: {
+                        a: {
+                            textDecoration: "none",
```

Comment:
I'm not a big fan of this as it leaves users with no visual indication of hyperlinks, and makes life difficult if we don't use other colours to distinguish.

Reply:
Can we revert the changes in `tailwind.config.js`?

Reply:
I'll assign @Hackin7 to work on this too, since it is related to https://github.com/AdvisorySG/ghost-advisory-theme/issues/309.